airflow/api/common/experimental/mark_tasks.py

```diff
@@ -49,6 +49,72 @@ def _create_dagruns(dag, execution_dates, state, run_id_template):
     return drs
 
 
+def _verify_tree(dag, dates, state):
```

Comment:
K. It is in line with DagRun.verify_integrity it might even move there in the future.